src/store.ts

```diff
@@ -110,9 +110,9 @@ const store: StoreOptions<RootState> = {
         findWallet: (state) => (id: string): WalletInfo | undefined => {
             return state.wallets.find((wallet) => wallet.id === id);
         },
-        findWalletByAddress: (state) => (address: string): WalletInfo | undefined => {
+        findWalletByAddress: (state) => (address: string, includeContracts: boolean): WalletInfo | undefined => {
```

Comment:
I made the new argument non-optional on purpose, so you _have to_ think about if you want contracts or not. Should reduce dev errors.